client/src/components/Results/Results.js

```diff
@@ -186,8 +186,8 @@ export default Results;
 //   const data = await nodeQueryBuilder(dataMap.category, dataMap.parent);
```

Comment:
We could probably get rid of all this commented code.

Reply:
Done